platform/android/export/export.cpp

```diff
@@ -744,6 +744,49 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+	// Implementation of EditorExportSaveSharedObject.
+	// Ignoring this setting allows all .so files to be included in gradle project
+	static Error ignore_so_file(void *p_userdata, const SharedObject &p_so) {
+		return OK;
+	}
+
+	// Writes p_data into a file at p_path, creating directories if necessary.
+	static Error store_file_at_path(const String &p_path, const Vector<uint8_t> &p_data, int compression_method = Z_DEFLATED) {
+		String dir = p_path.get_base_dir();
+		if (!DirAccess::exists(dir)) {
+			DirAccess *filesystem_da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
+			filesystem_da->make_dir_recursive(dir);
+			memdelete(filesystem_da);
+		}
+		FileAccess *fa = FileAccess::open(p_path, FileAccess::WRITE);
+		fa->store_buffer(p_data.ptr(), p_data.size());
+		ERR_FAIL_COND_V_MSG(!fa, ERR_CANT_CREATE, "Cannot create file '" + p_path + "'.");
+		memdelete(fa);
+		return OK;
+	}
+
+	// Writes string p_data to a file at p_path, creating directories if necessary.
+	static Error store_string_at_path(const String &p_path, const String &p_data, int compression_method = Z_DEFLATED) {
+		String dir = p_path.get_base_dir();
+		if (!DirAccess::exists(dir)) {
+			DirAccess *filesystem_da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
+			filesystem_da->make_dir_recursive(dir);
+			memdelete(filesystem_da);
+		}
+		FileAccess *fa = FileAccess::open(p_path, FileAccess::WRITE);
+		fa->store_string(p_data);
+		ERR_FAIL_COND_V_MSG(!fa, ERR_CANT_CREATE, "Cannot create file '" + p_path + "'.");
+		memdelete(fa);
+		return OK;
+	}
+
+	// Implementation of EditorExportSaveFunction.
```

Comment:
Can you clarify what this method is doing, and what its intended purpose will be.